### PR TITLE
[r] Refactor current code base

### PR DESCRIPTION
**Pull request by catriuspham**
This PR is highly about refactoring.

#### What have been done

- Separated integration tests into smaller parts
- Removed redundant tests (test_instance.py)
- Applied flake8
- Optimized imports
- Optimized code styles, indentations
- Use `is` in boolean and None comparison
- Noqa all F403 (`from x import *`)

### CODE REVIEW

**Comment by catriuspham** [comment]
@hieueastagile @NhanHo Please take a look

**Comment by NhanHo** [comment]
@catriuspham 
E712: `foo is True` or `foo is False`
E711: `foo is None`

**Comment by hieueastagile** [comment]
I guess that we should fix it as:
```python
x is True
x is False
x is None
```
For the import *, please check if it's needed or not, if not, just import all of them, if it's needed, just add a `# NOQA` to let flake8 ignore it.

**Comment by hieueastagile** [comment]
LGTM

**Comment by NhanHo** [comment]
LGTM

File: robber/expect.py [hunk 1]
```diff
@@ -61,8 +60,8 @@ def registered(cls, name):
             return True
 
     def __setup_chaining(self):
-        self.to   = self
-        self.be   = self
-        self.a    = self
-        self.an   = self
+        self.to = self
```
Comment (oyster) [inline robber/expect.py:63]: Do we need to format these? I think the current one is better.

**Pull request merged**